koku/masu/database/sql/reporting_ocpall_compute_summary_pt.sql

```diff
@@ -0,0 +1,66 @@
+-- OCP ON ALL COMPUTE SUMMARY PROCESSING
```

Comment:
suggestion: With AWS and OCP, we put the partable sql into folders within the `koku/masu/database/sql` directory. Can we move these `ocpall` sql files into `koku/masu/database/sql/openshift/all`?